htdocs/core/tpl/extrafields_list_array_fields.tpl.php

```diff
@@ -1,24 +1,27 @@
 <?php
-
-print '<!-- extrafields_list_array_fields.tpl.php -->'."\n";
+// do not use print in this tpl because page has not started
+// print '<!-- extrafields_list_array_fields.tpl.php -->'."\n";
```

Comment:
A tpl is by default data we want to output, the reste of page will output data, so output should already have started when we include a tpl.

Reply:
so we need to create another actions_list_array_fields.inc.php and empty this file ?

Reply:
Sorry. i don't understand. What i say is that the goal of file extrafields_list_array_fields.tpl.php is to output data.
So we can have print, in fact, we must have print inside.

Reply:
so filename is just extrafields_list_array_fields.inc.php ?

Reply:
Hum, i see. it seems this tpl is a third case. We already have tpl for shared code for action with filename "actions_..."
then for output with filename  "xxxx_..." with xxx that is not action and here it is a tpl for initialisation of vars (so not for output of data and to be executed before the action). This is the reason of my confusion.
So a good name would be "init_...". When a tpl start with init, we know that, like for "action..." we must not make output.

Never mind, i fixed this by doing your first PR suggestion (removing the print). But using a name with init_ for tpl that are not action and not output for future version will make things easier to understand.